meta-xt-driver-domain/recipes-extended/block/block.bb

```diff
@@ -17,10 +17,6 @@ RDEPENDS:${PN} += " \
     xen-tools-xenstore \
 "
 
-RRECOMMENDS:${PN} += " \
```

Comment:
In the new version of xen virtual/xenstored was renamed to virtual-xenstored. 
Here is how it was done in my repo:
```
--- /dev/null
+++ b/recipes-connectivity/xen-network/xen-network.bbappend
@@ -0,0 +1,5 @@
+RRECOMMENDS:${PN}:remove = "virtual/xenstored"
+
+RRECOMMENDS:${PN} += " \
+    virtual-xenstored \
+"

```

Reply:
Thanks. You are right, this works OK with the scarthgap. But we have an issue that meta-xt-common should work with kirkstone and scarthgap. So we should support both `virtual-xenstored` and `virtual/xenstored` depending on the yocto release used. That may be achieved by using some compile-time checks.
But at some point I found that `RRECOMMENDS` should not be used here at all, because
1. RRECOMMENDS specifies some packages, that are nice-to-have and extend functionality. And this is not the our case, because xenstore is a critical component. So we should use RDEPENDS.
2. RDEPENDS is already specified in all modified recipes.

So, I found that the best solution is to remove RRECOMMENDS.